acceptance-tests/README.md

```diff
@@ -221,25 +218,4 @@ objects are put into the `pages` subfolder.
 
 * Q: I'm getting `Error: Could not find or load main class`
 
-  A: Please verify version of sbt in your computer using `sbt sbtVersion` but run it not from the project root.
-
-
-* Q:  `value seconds is not a member of Int`
-
-  A:
-
-  ```
-  import scala.concurrent.duration._
-  ```
-
-* Q:
-  ```
-  type mismatch;
-   found   : String("Parent Group")
-   required: eu.timepit.refined.api.Refined[String,eu.timepit.refined.collection.NonEmpty]
-  ```
-
-  A:
-  ```
-  import eu.timepit.refined.auto._
-  ```
+  A: Please verify version of sbt in your computer using `sbt sbtVersion` but run it not from the project root.
```

Comment:
I'm wondering if this really matters? I think any sbt version is fine to use, unless it is something from many years ago.

Reply:
Yes, probably it doesn't matter. Cannot remind myself why I didn't even put this here 🤔 Definitely, something we don't need anymore.